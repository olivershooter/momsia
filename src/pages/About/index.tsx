import { Typography } from '@mui/material';
import React from 'react';

const About = () => {
  return <Typography sx={{ color: 'black' }}>about</Typography>;
};

export default About;
