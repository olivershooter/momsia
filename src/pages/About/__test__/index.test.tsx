import { BrowserRouter } from 'react-router-dom';
import { describe, expect, it } from 'vitest';
import About from '..';
import React from 'react';
import { render } from '@testing-library/react';

describe('About page', () => {
  it('renders and matches the snapshot', () => {
    const { asFragment } = render(
      <BrowserRouter>
        <About />
      </BrowserRouter>,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
