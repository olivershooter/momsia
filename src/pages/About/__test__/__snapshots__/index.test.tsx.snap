// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`About page > renders and matches the snapshot 1`] = `
<DocumentFragment>
  <p
    class="MuiTypography-root MuiTypography-body1 css-nnsfc0-MuiTypography-root"
  >
    about
  </p>
</DocumentFragment>
`;
